### PR TITLE
Reduces extinguisher violence

### LINKED ISSUE #22921: Trying to refill fire extinguishers with a water tank makes you hit the watertank
Well, as the title says, whenever i try to fill an extinguisher with a water tank i end up hiting the water tank which damages it so it ends up being destroyed before i can bless the whole station as the chaplain.

(note ive tried both with normal water and holy water and the bug still persists
watertank

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
fix: Nanotrasen decided that the "violent osmosis" method for refilling fire extinguishers was, while cathartic, too expensive, due to the water tank repair bills. Water tanks now have a tap.
fix: (refilling extinguishers from tanks won't make you hit them)
/:cl:

Fixes #22921

### CODE REVIEW

File: code/game/objects/items/weapons/extinguisher.dm [hunk 1]
```diff
@@ -66,7 +74,7 @@
 /obj/item/weapon/extinguisher/proc/AttemptRefill(atom/target, mob/user)
 	if(istype(target, /obj/structure/reagent_dispensers/watertank) && target.Adjacent(user))
 		var/safety_save = safety
-		safety = 1
+		safety = TRUE
```
Comment (Cyberboss) [inline code/game/objects/items/weapons/extinguisher.dm]: I personally think you should have to turn the safety on yourself before refilling it. What if the water tank is on fire?

Comment (XDTM) [inline code/game/objects/items/weapons/extinguisher.dm]: It blocks the water, so it's pointless to extinguish it.

**Comment by PKPenguin321** [comment]
Changelog doesn't explain the change in plain english well enough. Just from reading that I have no idea what you're talking about.

**Comment by XDTM** [comment]
there you go
